### DISCUSSION TITLE
Got a idea - about gcc

### DISCUSSION THREAD

**Discussion by saswatsamal** [hackclubiter/onEditor#6] [Ideas]
- GCC is a cross compiler and that nearly achieves the functionality you look for - [How to Build a GCC Cross-Compiler](https://preshing.com/20141119/how-to-build-a-gcc-cross-compiler/)

- Also got an useful discussion on this on [Stackoverflow](https://softwareengineering.stackexchange.com/questions/165543/how-to-write-a-very-basic-compiler#answer-165558)

**Comment by saswatsamal** [comment]
A website which works on various languages- [Ideone](https://ideone.com/)

**Comment by saswatsamal** [comment]
An quora answer explaining how to get started: [Quora Link](https://qr.ae/pG8IDd)

**Comment by Saswat7101** [comment]
This is quite elaborate but most of the things went over my head 😂

**Reply by LoopGlitch26** [reply]
It's alright, we'll understand eventually, if not immediately ;)
